src/csharp/Microsoft.Spark.UnitTest/Sql/TimestampTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql.Types;
+using Xunit;
+
+namespace Microsoft.Spark.UnitTest
+{
+    public class TimestampTests
+    {
+        [Fact]
+        public void TimestampTest()
+        {
+            {
+                var timestamp = new Timestamp(
+                    new DateTime(2020, 1, 2, 15, 30, 30, DateTimeKind.Utc));
+
+                // Validate values.
+                Assert.Equal(2020, timestamp.Year);
+                Assert.Equal(1, timestamp.Month);
+                Assert.Equal(2, timestamp.Day);
+                Assert.Equal(15, timestamp.Hour);
+                Assert.Equal(30, timestamp.Minute);
+                Assert.Equal(30, timestamp.Second);
+                Assert.Equal(0, timestamp.Microsecond);
+
+                // Validate ToString().
+                Assert.Equal("2020-01-02 15:30:30.000000", timestamp.ToString());
+
+                // Validate ToDateTime().
+                Assert.IsType<DateTime>(timestamp.ToDateTime());
```

Comment:
we need to verify the returned value is same is the one you started with.

Reply:
I think we also need to verify that even if you pass in non-UTC `DateTime`, you get the UTC one.

Reply:
Added the tests, thanks!